globals.go

```diff
@@ -213,9 +213,29 @@ const (
 	// app server should log to stdout.
 	standardOutputLoggingOverride = "STDOUT"
 
-	DefaultTaskActivator   = ""
-	StepbackTaskActivator  = "stepback"
+	// DefaultTaskActivator is a deprecated legacy activator that used
+	// to be a majority of non-stepback and non-API activations.
+	DefaultTaskActivator = ""
+	// APIServerTaskActivator represents Evergreen's internal API activator
 	APIServerTaskActivator = "apiserver"
+	// StepbackTaskActivator represents the activator for tasks activated
+	// due to stepback.
+	StepbackTaskActivator = "stepback"
```

Comment:
would it be chaotic to have all of these say activator at the end? Or does that seem redundant for the use cases where we see these?

Reply:
I think it's a good suggestion, it'll help to be more explicit with our naming and make it clear that the only non-users that activate things in Evergreen must be "activators". Added